tensorflow/lite/delegates/nnapi/nnapi_delegate.cc

```diff
@@ -2023,7 +2023,7 @@ bool NNAPIDelegateKernel::Validate(
              "NNAPI only supports constant int32 axis tensor.", &val_ctx);
     } break;
     case kTfLiteBuiltinSplit: {
-      ExpectOpVersion(version, 1, &val_ctx);
+      ExpectOpVersion(version, 3, &val_ctx);
```

Comment:
Do you need to filter out this case when running on < 1.2 NNAPI?

Reply:
Yes, thanks for the review.